contrib/endpoints/src/api_manager/quota_control.cc

```diff
@@ -46,7 +46,10 @@ void QuotaControl(std::shared_ptr<context::RequestContext> context,
         TRACE(trace_span) << "Quota service control request returned with "
                           << "status " << status.ToString();
 
-        continuation(status);
+        // if the quota control service API is not available, then pass client
+        // requests
+        continuation((status.code() == Code::UNAVAILABLE) ? utils::Status::OK
```

Comment:
In Aggregated::Quota, we call Proto::ConvertAllocateQuotaResponse when the response of the Call is OK.

This is to handle Code::UNAVAILABLE from Aggregated::Call